packages/drivers/debugger/src/messageSchema.ts

```diff
@@ -334,15 +334,15 @@ const mergeTreeDeltaOpSchema = {
     properties: {
         type: {
             type: "number",
-            minimum: MergeTreeDeltaType.INSERT,
-            maximum: MergeTreeDeltaType.ANNOTATE,
+            minimum: 0 /* MergeTreeDeltaType.INSERT */,
```

Comment:
Feels like we would benefit from MergeTreeDeltaType having min & max values in the enum, no?

Reply:
Yup, only removed here because of layering/publishing issues with the test tools.  #4704 tracks that refactoring and should revert this part